### DISCUSSION TITLE
VesselVio Analyze crashed when analyzing my .nii data

### DISCUSSION THREAD

**Discussion by Jamie790** [JacobBumgarner/VesselVio#74] [General]
### What happened?

Hi,

I load my vessel label data as .nii and then i click Analyze tab, the GUI crashed ad File Status (Analyzing features steps). Please help me with this issue.

Many thanks,

### Operating System

Windows

### App Build

Downloaded App

### Pseudocode or Screenshots

```python 
 Insert code here 
 ```
 Or drag your screenshot here!

**Comment by JacobBumgarner** [comment]
Hello, 
I’ll need a bit more detail. How did you create the .nii file? What settings did you use for the analysis? Can you share the nii file?

**Comment by Jamie790** [comment]
Hi, 
Can you share with me your email, i can share with you the data (It is research data so i can not share public).

**Comment by JacobBumgarner** [comment]
Hi, sure, please see my email in the **Other** section of the main page of the repository.
